tsconfig.json

```
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "noImplicitAny": false,
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "skipLibCheck": true,
+    "jsx": "preserve",
+    "target": "es2017",
```

```suggestion
    "target": "esnext",
```

Probably don't need any polyfills so I usually just keep this as esnext when targeting Node.

---

Yep, it should be `esnext`.

BTW, it confused me that VSCode keep warning about `Object.entries()`, while compiling with tsc is fine. I tried various ways to fix it, none of them works. So I end up use `// @ts-ignore` for this https://github.com/badgen/badgen.net/pull/282/files#diff-577c40df72d7574b39845c9931c83104R5 😕